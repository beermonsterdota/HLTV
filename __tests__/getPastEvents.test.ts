import HLTV from '../src/index.js'
import { EventType } from '../src/shared/EventType.js'
import { sleep } from '../src/utils.js'
import { expect, test } from 'vitest'

test('getPastEvents', async () => {
  await sleep(3000)
  expect(
    await HLTV.getPastEvents({
      startDate: '2017-01-01',
      endDate: '2017-12-31',
      eventType: EventType.InternationalLAN,
      delayBetweenPageRequests: 3000
    })
  ).toMatchSnapshot()
  await sleep(3000)
}, 30000)
