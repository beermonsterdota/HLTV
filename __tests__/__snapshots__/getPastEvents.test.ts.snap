// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`getPastEvents 1`] = `
[
  {
    "dateEnd": 1514113200000,
    "dateStart": 1513854000000,
    "id": 3464,
    "location": {
      "code": "RS",
      "name": "Belgrade, Serbia",
    },
    "name": "Winner of the Future 2017",
    "numberOfTeams": 8,
    "prizePool": "$4,725",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1513508400000,
    "dateStart": 1513335600000,
    "id": 3313,
    "location": {
      "code": "MX",
      "name": "Cancun, Mexico",
    },
    "name": "ECS Season 4 Finals",
    "numberOfTeams": 8,
    "prizePool": "$660,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1512903600000,
    "dateStart": 1512471600000,
    "id": 3072,
    "location": {
      "code": "DK",
      "name": "Odense, Denmark",
    },
    "name": "ESL Pro League Season 6 Finals",
    "numberOfTeams": 12,
    "prizePool": "$750,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1512817200000,
    "dateStart": 1512558000000,
    "id": 3283,
    "location": {
      "code": "MY",
      "name": "Kuala Lumpur, Malaysia",
    },
    "name": "ROG MASTERS 2017 Grand Finals",
    "numberOfTeams": 6,
    "prizePool": "$235,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1512817200000,
    "dateStart": 1507370400000,
    "id": 3201,
    "location": {
      "code": "GB",
      "name": "London, United Kingdom",
    },
    "name": "Gfinity Elite Series Season 2",
    "numberOfTeams": 8,
    "prizePool": "$110,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1512298800000,
    "dateStart": 1512126000000,
    "id": 2574,
    "location": {
      "code": "SE",
      "name": "Jönköping, Sweden",
    },
    "name": "DreamHack Open Winter 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1511694000000,
    "dateStart": 1511607600000,
    "id": 3224,
    "location": {
      "code": "BY",
      "name": "Minsk, Belarus",
    },
    "name": "FCDB Cup 2017",
    "numberOfTeams": 8,
    "prizePool": "$40,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1511694000000,
    "dateStart": 1511434800000,
    "id": 3115,
    "location": {
      "code": "ES",
      "name": "Barcelona, Spain",
    },
    "name": "WESG 2017 Europe & CIS Regional Finals Female",
    "numberOfTeams": 8,
    "prizePool": "$16,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1511694000000,
    "dateStart": 1511434800000,
    "id": 3114,
    "location": {
      "code": "ES",
      "name": "Barcelona, Spain",
    },
    "name": "WESG 2017 Europe & CIS Regional Finals",
    "numberOfTeams": 20,
    "prizePool": "$92,500",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1511607600000,
    "dateStart": 1511521200000,
    "id": 2998,
    "location": {
      "code": "DK",
      "name": "Copenhagen, Denmark",
    },
    "name": "BLAST Pro Series Copenhagen 2017",
    "numberOfTeams": 6,
    "prizePool": "$250,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1511434800000,
    "dateStart": 1511089200000,
    "id": 3292,
    "location": {
      "code": "CN",
      "name": "Shenzhen, China",
    },
    "name": "China Top 2017 Shenzhen",
    "numberOfTeams": 4,
    "prizePool": "$113,361",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1511089200000,
    "dateStart": 1510657200000,
    "id": 2956,
    "location": {
      "code": "US",
      "name": "Burbank & Oakland, California, USA",
    },
    "name": "IEM Oakland 2017",
    "numberOfTeams": 12,
    "prizePool": "$300,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1511002800000,
    "dateStart": 1510916400000,
    "id": 3355,
    "location": {
      "code": "KR",
      "name": "Busan, South Korea",
    },
    "name": "WEGL 2017 Super Fight Invitational",
    "numberOfTeams": 4,
    "prizePool": "$53,580",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1511002800000,
    "dateStart": 1510916400000,
    "id": 3166,
    "location": {
      "code": "AU",
      "name": "Sydney, Australia",
    },
    "name": "ZEN League Season 2 Finals",
    "numberOfTeams": 4,
    "prizePool": "$24,300",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1510484400000,
    "dateStart": 1510398000000,
    "id": 3165,
    "location": {
      "code": "US",
      "name": "Santa Ana, California, USA",
    },
    "name": "iBUYPOWER Masters 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1510398000000,
    "dateStart": 1510138800000,
    "id": 3069,
    "location": {
      "code": "KR",
      "name": "Busan, South Korea",
    },
    "name": "IeSF World Championship 2017",
    "numberOfTeams": 16,
    "prizePool": "$17,978",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1509879600000,
    "dateStart": 1509620400000,
    "id": 3252,
    "location": {
      "code": "CA",
      "name": "Toronto, Canada",
    },
    "name": "Americas Minor - ELEAGUE Major 2018",
    "numberOfTeams": 8,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1509879600000,
    "dateStart": 1509620400000,
    "id": 3251,
    "location": {
      "code": "RO",
      "name": "Bucharest, Romania",
    },
    "name": "Europe Minor - ELEAGUE Major 2018",
    "numberOfTeams": 8,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1509879600000,
    "dateStart": 1509620400000,
    "id": 3190,
    "location": {
      "code": "CN",
      "name": "Shanghai, China",
    },
    "name": "SL i-League Invitational Shanghai 2017",
    "numberOfTeams": 8,
    "prizePool": "$150,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1509271200000,
    "dateStart": 1509012000000,
    "id": 3250,
    "location": {
      "code": "RO",
      "name": "Bucharest, Romania",
    },
    "name": "CIS Minor - ELEAGUE Major 2018",
    "numberOfTeams": 8,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1509271200000,
    "dateStart": 1509012000000,
    "id": 3249,
    "location": {
      "code": "KR",
      "name": "Seoul, South Korea",
    },
    "name": "Asia Minor - ELEAGUE Major 2018",
    "numberOfTeams": 8,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1509271200000,
    "dateStart": 1508839200000,
    "id": 3133,
    "location": {
      "code": "RU",
      "name": "Saint Petersburg, Russia",
    },
    "name": "EPICENTER 2017",
    "numberOfTeams": 8,
    "prizePool": "$490,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1509184800000,
    "dateStart": 1509098400000,
    "id": 3204,
    "location": {
      "code": "ES",
      "name": "Madrid, Spain",
    },
    "name": "ESL Clash of Nations",
    "numberOfTeams": 8,
    "prizePool": "$10,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1508752800000,
    "dateStart": 1508752800000,
    "id": 3153,
    "location": {
      "code": "RU",
      "name": "Saint Petersburg, Russia",
    },
    "name": "EPICENTER 2017 Wild Card Qualifier",
    "numberOfTeams": 4,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1508666400000,
    "dateStart": 1508493600000,
    "id": 2573,
    "location": {
      "code": "US",
      "name": "Denver, Colorado, USA",
    },
    "name": "DreamHack Open Denver 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1508666400000,
    "dateStart": 1508407200000,
    "id": 2921,
    "location": {
      "code": "CN",
      "name": "Shanghai, China",
    },
    "name": "eXTREMESLAND 2017 Asia Finals",
    "numberOfTeams": 16,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1508061600000,
    "dateStart": 1507975200000,
    "id": 3183,
    "location": {
      "code": "DE",
      "name": "Krefeld, Germany",
    },
    "name": "ROG MASTERS 2017 EMEA Finals",
    "numberOfTeams": 8,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1507888800000,
    "dateStart": 1504864800000,
    "id": 3016,
    "location": {
      "code": "US",
      "name": "Atlanta, Georgia, USA",
    },
    "name": "ELEAGUE CS:GO Premier 2017",
    "numberOfTeams": 16,
    "prizePool": "$1,000,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1507370400000,
    "dateStart": 1507197600000,
    "id": 3101,
    "location": {
      "code": "DE",
      "name": "Husum, Germany",
    },
    "name": "NGC Masters 2017",
    "numberOfTeams": 8,
    "prizePool": "$17,585",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1506852000000,
    "dateStart": 1506679200000,
    "id": 3189,
    "location": {
      "code": "CN",
      "name": "Kunshan, China",
    },
    "name": "China Top 2017 Kunshan",
    "numberOfTeams": 4,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1506247200000,
    "dateStart": 1506160800000,
    "id": 3046,
    "location": {
      "code": "US",
      "name": "Burbank, California, USA",
    },
    "name": "ESEA Season 25 MDL Global Challenge",
    "numberOfTeams": 8,
    "prizePool": "$50,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1505642400000,
    "dateStart": 1505469600000,
    "id": 2636,
    "location": {
      "code": "US",
      "name": "New York, New York, USA",
    },
    "name": "ESL One New York 2017",
    "numberOfTeams": 8,
    "prizePool": "$250,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1505037600000,
    "dateStart": 1504864800000,
    "id": 2973,
    "location": {
      "code": "DK",
      "name": "Sønderborg, Denmark",
    },
    "name": "Cross Border Esport 2017",
    "numberOfTeams": 24,
    "prizePool": "$15,300",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1505037600000,
    "dateStart": 1504864800000,
    "id": 2572,
    "location": {
      "code": "CA",
      "name": "Montreal, Canada",
    },
    "name": "DreamHack Open Montreal 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1505037600000,
    "dateStart": 1504778400000,
    "id": 3061,
    "location": {
      "code": "GR",
      "name": "Mykonos, Greece",
    },
    "name": "ESG Tour Mykonos 2017",
    "numberOfTeams": 8,
    "prizePool": "$235,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1504432800000,
    "dateStart": 1504346400000,
    "id": 3140,
    "location": {
      "code": "MO",
      "name": "Ilhas, Macau",
    },
    "name": "GIRLGAMER Esports Festival 2017",
    "numberOfTeams": 4,
    "prizePool": "$12,781",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1504432800000,
    "dateStart": 1504087200000,
    "id": 2684,
    "location": {
      "code": "SE",
      "name": "Malmö, Sweden",
    },
    "name": "DreamHack Masters Malmö 2017",
    "numberOfTeams": 16,
    "prizePool": "$250,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1504346400000,
    "dateStart": 1504346400000,
    "id": 3085,
    "location": {
      "code": "DE",
      "name": "Peine, Germany",
    },
    "name": "iiyama CS:GO HomeMasters #3 Finals",
    "numberOfTeams": 2,
    "prizePool": "$3,569",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1504346400000,
    "dateStart": 1499508000000,
    "id": 2979,
    "location": {
      "code": "GB",
      "name": "London, United Kingdom",
    },
    "name": "Gfinity Elite Series Season 1",
    "numberOfTeams": 8,
    "prizePool": "$110,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1503828000000,
    "dateStart": 1503741600000,
    "id": 3093,
    "location": {
      "code": "RU",
      "name": "Moscow, Russia",
    },
    "name": "Binary Dragons Prestige LAN",
    "numberOfTeams": 8,
    "prizePool": "$10,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1500804000000,
    "dateStart": 1500631200000,
    "id": 2571,
    "location": {
      "code": "US",
      "name": "Atlanta, Georgia, USA",
    },
    "name": "DreamHack Open Atlanta 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1500112800000,
    "dateStart": 1499940000000,
    "id": 2570,
    "location": {
      "code": "ES",
      "name": "Valencia, Spain",
    },
    "name": "DreamHack Open Valencia 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1499594400000,
    "dateStart": 1499162400000,
    "id": 2635,
    "location": {
      "code": "DE",
      "name": "Cologne, Germany",
    },
    "name": "ESL One Cologne 2017",
    "numberOfTeams": 16,
    "prizePool": "$250,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1498989600000,
    "dateStart": 1498730400000,
    "id": 2721,
    "location": {
      "code": "RO",
      "name": "Bucharest, Romania",
    },
    "name": "PGL Major Krakow 2017 Main Qualifier",
    "numberOfTeams": 16,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1498384800000,
    "dateStart": 1498212000000,
    "id": 2763,
    "location": {
      "code": "GB",
      "name": "London, United Kingdom",
    },
    "name": "ECS Season 3 Finals",
    "numberOfTeams": 8,
    "prizePool": "$660,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1498298400000,
    "dateStart": 1498298400000,
    "id": 2890,
    "location": {
      "code": "DE",
      "name": "Hamburg, Germany",
    },
    "name": "Legend Series #2 Finals",
    "numberOfTeams": 4,
    "prizePool": "$3,360",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1497866400000,
    "dateStart": 1497693600000,
    "id": 2924,
    "location": {
      "code": "SE",
      "name": "Jönköping, Sweden",
    },
    "name": "ESU Masters 2017",
    "numberOfTeams": 8,
    "prizePool": "$11,468",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1497866400000,
    "dateStart": 1497693600000,
    "id": 2569,
    "location": {
      "code": "SE",
      "name": "Jönköping, Sweden",
    },
    "name": "DreamHack Open Summer 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1497780000000,
    "dateStart": 1497780000000,
    "id": 2889,
    "location": {
      "code": "RU",
      "name": "Moscow, Russia",
    },
    "name": "Adrenaline Cyber League 2017",
    "numberOfTeams": 4,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1497780000000,
    "dateStart": 1497693600000,
    "id": 2884,
    "location": {
      "code": "AU",
      "name": "Sydney, Australia",
    },
    "name": "ZEN Esports Network League Season 1 Finals",
    "numberOfTeams": 4,
    "prizePool": "$37,184",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1497780000000,
    "dateStart": 1497520800000,
    "id": 2723,
    "location": {
      "code": "RO",
      "name": "Bucharest, Romania",
    },
    "name": "Europe Minor - PGL Major Krakow 2017",
    "numberOfTeams": 8,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1497607200000,
    "dateStart": 1497607200000,
    "id": 2853,
    "location": {
      "code": "US",
      "name": "Atlanta, Georgia, USA",
    },
    "name": "ELEAGUE Clash for Cash",
    "numberOfTeams": 2,
    "prizePool": "$250,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1497175200000,
    "dateStart": 1496916000000,
    "id": 2724,
    "location": {
      "code": "RU",
      "name": "Moscow, Russia",
    },
    "name": "CIS Minor - PGL Major Krakow 2017",
    "numberOfTeams": 8,
    "prizePool": "$50,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1497175200000,
    "dateStart": 1496916000000,
    "id": 2722,
    "location": {
      "code": "US",
      "name": "Santa Ana, California, USA",
    },
    "name": "Americas Minor - PGL Major Krakow 2017",
    "numberOfTeams": 8,
    "prizePool": "$50,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1496570400000,
    "dateStart": 1496311200000,
    "id": 2725,
    "location": {
      "code": "CN",
      "name": "Beijing, China",
    },
    "name": "Asia Minor - PGL Major Krakow 2017",
    "numberOfTeams": 8,
    "prizePool": "$50,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1496570400000,
    "dateStart": 1496138400000,
    "id": 2557,
    "location": {
      "code": "US",
      "name": "Dallas, Texas, USA",
    },
    "name": "ESL Pro League Season 5 Finals",
    "numberOfTeams": 12,
    "prizePool": "$750,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1495965600000,
    "dateStart": 1495879200000,
    "id": 2777,
    "location": {
      "code": "PL",
      "name": "Wroclaw, Poland",
    },
    "name": "GeForce Cup 2017",
    "numberOfTeams": 8,
    "prizePool": "$30,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1495879200000,
    "dateStart": 1495879200000,
    "id": 2834,
    "location": {
      "code": "RS",
      "name": "Belgrade, Serbia",
    },
    "name": "The Adria Masters Season 1 Finals",
    "numberOfTeams": 4,
    "prizePool": "$5,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1494756000000,
    "dateStart": 1494669600000,
    "id": 2801,
    "location": {
      "code": "RO",
      "name": "Bucharest, Romania",
    },
    "name": "HTC 2v2 Invitational by PGL",
    "numberOfTeams": 8,
    "prizePool": "$50,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1494756000000,
    "dateStart": 1494669600000,
    "id": 2749,
    "location": {
      "code": "GB",
      "name": "Leicester, United Kingdom",
    },
    "name": "ESEA Season 24 MDL Global Challenge",
    "numberOfTeams": 8,
    "prizePool": "$50,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1494237600000,
    "dateStart": 1494064800000,
    "id": 2568,
    "location": {
      "code": "FR",
      "name": "Tours, France",
    },
    "name": "DreamHack Open Tours 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1494151200000,
    "dateStart": 1494064800000,
    "id": 2807,
    "location": {
      "code": "PT",
      "name": "Lisbon, Portugal",
    },
    "name": "4Gamers CS:GO Masters",
    "numberOfTeams": 6,
    "prizePool": "$10,997",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1494151200000,
    "dateStart": 1493805600000,
    "id": 2713,
    "location": {
      "code": "AU",
      "name": "Sydney, Australia",
    },
    "name": "IEM Sydney 2017",
    "numberOfTeams": 8,
    "prizePool": "$200,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1493546400000,
    "dateStart": 1493373600000,
    "id": 2567,
    "location": {
      "code": "US",
      "name": "Austin, Texas, USA",
    },
    "name": "DreamHack Open Austin 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1492941600000,
    "dateStart": 1492682400000,
    "id": 2641,
    "location": {
      "code": "US",
      "name": "Los Angeles, California",
    },
    "name": "cs_summit Spring 2017",
    "numberOfTeams": 8,
    "prizePool": "$150,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1492250400000,
    "dateStart": 1492164000000,
    "id": 2753,
    "location": {
      "code": "DK",
      "name": "Copenhagen, Denmark",
    },
    "name": "Copenhagen Games 2017",
    "numberOfTeams": 16,
    "prizePool": "$56,300",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1492250400000,
    "dateStart": 1492077600000,
    "id": 2663,
    "location": {
      "code": "DK",
      "name": "Copenhagen, Denmark",
    },
    "name": "Copenhagen Games 2017 Female",
    "numberOfTeams": 28,
    "prizePool": "$15,025",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1492164000000,
    "dateStart": 1492077600000,
    "id": 2675,
    "location": {
      "code": "DK",
      "name": "Copenhagen, Denmark",
    },
    "name": "Copenhagen Games 2017 BYOC Qualifier",
    "numberOfTeams": 76,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1491732000000,
    "dateStart": 1491300000000,
    "id": 2683,
    "location": {
      "code": "UA",
      "name": "Kiev, Ukraine",
    },
    "name": "SL i-League StarSeries Season 3 Finals",
    "numberOfTeams": 16,
    "prizePool": "$300,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1491040800000,
    "dateStart": 1491040800000,
    "id": 2742,
    "location": {
      "code": "DE",
      "name": "Siek, Germany",
    },
    "name": "Legend Series #1 Finals",
    "numberOfTeams": 2,
    "prizePool": "$2,917",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1489921200000,
    "dateStart": 1489748400000,
    "id": 2160,
    "location": {
      "code": "CN",
      "name": "Chengde, China",
    },
    "name": "China Cup 2017",
    "numberOfTeams": 6,
    "prizePool": "$99,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1488711600000,
    "dateStart": 1488366000000,
    "id": 2637,
    "location": {
      "code": "PL",
      "name": "Katowice, Poland",
    },
    "name": "IEM Katowice 2017",
    "numberOfTeams": 12,
    "prizePool": "$250,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1488538800000,
    "dateStart": 1488452400000,
    "id": 2662,
    "location": {
      "code": "PL",
      "name": "Katowice, Poland",
    },
    "name": "Intel Challenge Katowice 2017",
    "numberOfTeams": 8,
    "prizePool": "$30,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1487502000000,
    "dateStart": 1487156400000,
    "id": 2541,
    "location": {
      "code": "US",
      "name": "Las Vegas, Nevada, USA",
    },
    "name": "DreamHack Masters Las Vegas 2017",
    "numberOfTeams": 16,
    "prizePool": "$450,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1486206000000,
    "dateStart": 1486119600000,
    "id": 2649,
    "location": {
      "code": "FI",
      "name": "Helsinki, Finland",
    },
    "name": "Assembly Winter 2017",
    "numberOfTeams": 16,
    "prizePool": "$15,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1484478000000,
    "dateStart": 1484305200000,
    "id": 2566,
    "location": {
      "code": "DE",
      "name": "Leipzig, Germany",
    },
    "name": "DreamHack ASTRO Open Leipzig 2017",
    "numberOfTeams": 8,
    "prizePool": "$100,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1484478000000,
    "dateStart": 1484218800000,
    "id": 2564,
    "location": {
      "code": "CN",
      "name": "Changzhou, China",
    },
    "name": "WESG 2016 World Finals",
    "numberOfTeams": 24,
    "prizePool": "$1,500,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1483873200000,
    "dateStart": 1483700400000,
    "id": 2538,
    "location": {
      "code": "US",
      "name": "Burbank, California, USA",
    },
    "name": "ESEA Season 23 MDL Global Challenge",
    "numberOfTeams": 8,
    "prizePool": "$50,000",
    "type": "INTLLAN",
  },
  {
    "dateEnd": 1483873200000,
    "dateStart": 1483614000000,
    "id": 2642,
    "location": {
      "code": "SE",
      "name": "Stockholm, Sweden",
    },
    "name": "ASUS ROG Community Challenge Grand Finals 2017",
    "numberOfTeams": 8,
    "prizePool": "Other",
    "type": "INTLLAN",
  },
]
`;
